cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -84,7 +84,7 @@ In single file mode:
 In list file mode:
     The FILE_PATH should point to a a file listing multiple audio files
     with the format 'Utterance_ID \t FILE_PATH \n'.
-    Each entry should be on its own line.  
+    Each entry should be on its own line.
     Utterance_IDs may not contain whitespace.
 
```

Comment:
Would you mind adding a line to the `longMsg` field about supported file types?  Probably something like this:

```txt
CubicSvr supports the following file formats: [  .wav  .flac  .mp3  .vox  .raw  ].
This command automatically detects the file format based on the file extension.  
```

Reply:
done. can you check again if what I added ok OK?